cmd/agent/app/flags.go

```diff
@@ -47,8 +47,8 @@ var defaultProcessors = []struct {
 func AddFlags(flags *flag.FlagSet) {
 	for _, p := range defaultProcessors {
 		prefix := fmt.Sprintf("processor.%s-%s.", p.model, p.protocol)
-		flags.Int(prefix+suffixWorkers, defaultServerWorkers, "how many workers the processor should run")
-		flags.Int(prefix+suffixServerQueueSize, defaultQueueSize, "length of the queue for the UDP server")
+		flags.Int(prefix+suffixWorkers, defaultServerWorkers, "how many workers the processor should run, deprecated - use reporter config")
+		flags.Int(prefix+suffixServerQueueSize, defaultQueueSize, "length of the queue for the UDP server, deprecated - use reporter config")
```

Comment:
Could this be used to set the new option value in the reporter? `queue.memory.size`

Reply:
I did think about this earlier, but do we want to reuse existing flags to modify new behavior? Could that cause some unexpected user experience?

Although in short term we could remap this to the same one (use the new flag if that has value, otherwise check this one and log something).

Reply:
We did the same before for other flags, it prevented us from breaking changes. I think we also generated warning message if the old flag has been used.